server/index.js

```diff
@@ -54,7 +54,7 @@ app.patch('/overview/:campId/owner', async (req, res) => {
   res.send(mockData);
 });
 
-app.patch('/overview/:campId/pricing', async (req, res) => {
+app.get('/overview/:campId/pricing', async (req, res) => {
```

Comment:
I noticed you were using PATCH on a previous . Curious what made you switch to GET?!

Reply:
I'm not actually updating anything so It's just a GET.